pkg/network/tracer/tracer.go

```diff
@@ -341,10 +341,8 @@ func (t *Tracer) addProcessInfo(c *network.ConnectionStats) {
 	}
 
 	if len(p.Tags) > 0 {
-		c.Tags = make(map[*intern.Value]struct{}, len(p.Tags))
-		for _, t := range p.Tags {
-			c.Tags[t] = struct{}{}
-		}
+		c.Tags = make([]*intern.Value, len(p.Tags))
```

Comment:
For now I think this change is fine since this is the only place we are inserting into `c.Tags`. But I believe we will expand its use in the future, so maintaining a list of unique tags may not be worthwhile with a slice anymore.